multigeiger/display.cpp

```diff
@@ -28,25 +28,28 @@ void display_start_screen(void) {
   pu8x8->clear();
   if (isLoraBoard) {
     // Display is only 4 lines by 8 characters; lines counting from 2 to 5
-    pu8x8->setFont(u8x8_font_amstrad_cpc_extended_f);        // u8x8 can handle only 8x8, smaller font has no advantage
+    pu8x8->setFont(u8x8_font_victoriamedium8_r);
     for (int i = 2; i < 6; i++) {
       pu8x8->drawString(0, i, "        ");  // clear all 4 lines
     }
```

Comment:
as you have a lora board, can you check whether these 3 lines are really needed?

see line 28, that should already clear the display.

Reply:
that is what #208 was about, iirc.

Reply:
and if they actually **are** needed, we have a clearLine (or so) function that should be called for this.

Reply:
No, works fine without the extra lines. Removed them.